Gruntfile.js

```diff
@@ -120,7 +114,7 @@ module.exports = function(grunt){
   });
 
   [
-    'grunt-contrib-copy',
+    'grunt-sync',
     'grunt-contrib-watch',
     'grunt-contrib-clean',
```

Comment:
There's still a reference to grunt-contrib-clean - which is no longer required, right?